src/pathwayvis/components/map/map.component.ts

```diff
@@ -520,6 +537,19 @@ class MapComponentCtrl {
       this.processActionClick(this.contextActions[1], this.contextElement));
   }
 
+  public handleObjectiveDirection(args) {
+    const tooltipContainer = d3.select('div#tooltip-container');
+    if (this.contextActions[4].type === 'reaction:maximize:do') {
```

Comment:
This is code duplication from https://github.com/DD-DeCaF/metabolica-ui-map/pull/155/files#diff-62c06d0a48047fe88322f20be6e856e2R51